src/database/config.js

```diff
@@ -0,0 +1,21 @@
+import pg from "pg";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const isProduction = process.env.NODE_ENV === "production";
+
+/**
+ * Use this pattern in naming your local connection strings
+ *  postgres://{database_username}:{database_password}@{host}:{port}/{database_name}
+ */
+
+const connectionString = isProduction
+  ? process.env.DATABASE_URL
+  : "postgres://postgres:juninho1@127.0.0.1:5432/weTravel";
+
+const { Pool } = pg;
```

Comment:
Wow, this is cool and straight forward. But what if you do like this from line 1: 
#### import { Pool } from 'pg'; ?

Reply:
line 17 makes it redundant!

Reply:
Yes that makes sense. thanks man

Reply:
Yep. We got each others back!

Reply:
@UggoPrince good suggestion. @ugwumadu116 Great work. however, i'll suggest you try running the code locally, just to confirm if you're able to connect to the DB?

Reply:
Thanks for that @voke1 but what it does is to create a connection link that the services in the app will use to connect to the database any time they want to communicate with it.